signwriting_transcription/pose_to_signwriting/Online_bash_file.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+# Clone the repository
+git clone https://github.com/sign-language-processing/signwriting-transcription.git
+cd signwriting-transcription
+
+# Install the required packages
+pip install .[dev,pose_to_signwriting]
+pip install ruamel.yaml
+
+# Download and unzip the sign-vq.zip file
+wget 'https://drive.usercontent.google.com/download?id=1V_Af2oqY28QgkE1e8jZzEvuxf6N3bLkB&export=download&authuser=0&confirm=t&uuid=c134a3b0-59c6-4279-b3d9-e918523fd913&at=APZUnTXILGGtZIF-UVbFYXHddcFX%3A1718897741079' -O sign-vq.zip
+unzip sign-vq.zip -d sign-vq
+cd sign-vq
+pip install .
+cd ..
```

Comment:
why is this not just `pip install git+https://github.com/sign-language-processing/sign-vq.git`?

Reply:
done, you right - there used to be reason for that but it seen ok.